wys/api/README.md

```diff
@@ -10,90 +16,85 @@ The script can request data on any day specified by the API call.
+As described above, `wys.speed_counts_agg_5kph` has data aggregated to 1-hour and 5 km/h bins using the `aggregate_speed_counts_one_hour_5kph()` function. Values for the speed bins are replaced by lookup table IDs.
 
-|Field name|Data type|Description|Example|
+|Field name|Type|Description|Example|
 |------|------|-------|------|
-`speed_counts_agg_5kph_id`|integer|A unique identifier for the `speed_counts_agg_5kph` table|2655075
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`datetime_bin`|timestamp|Start time of the 1 houraggregated bin|2018-10-29 10:00:00
-`speed_id`|integer|A unique identifier for the 5 kph speed bin in the `speed_bins` table|5
-`count`|integer|Number of vehicles in datetime_bin/api_id/speed bin combination|7
+|`speed_counts_agg_5kph_id`|bigint|A unique identifier for the `speed_counts_agg_5kph` table|2655075|
+|`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967|
+|`datetime_bin`|timestamp|Start time of the 1 houraggregated bin|2018-10-29 10:00:00|
+|`speed_id`|integer|A unique identifier for the 5 kph speed bin in the `speed_bins` table|5|
+|`volume`|integer|Number of vehicles in datetime_bin/api_id/speed bin combination|7|
 
 ### Lookup Tables
 
-`speed_bins` is a lookup table containing all the 5km/h speed bin. Bin number 25 contains any speed over 120km/h.
+`speed_bins_old` is a lookup table containing all the 5km/h speed bin. Bin number 21 contains any speed over 100km/h.
 
-|Field name|Data type|Description|Example|
+|Field name|Type|Description|Example|
 |------|------|-------|------|
 `speed_id`|integer|A unique identifier for the `speed_bins` table|5
 `speed_bin`|integer range|Range of speeds for each speed bin. The upper limit is not inclusive.|[10-25)
 
-`locations` contains each ID used for the API, direction of traffic the sign is gathering data for, address and name of the sign. Some signs may have information missing from any one of the fields.
-
-|Field name|Data type|Description|Example|
-|------|------|-------|------|
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`address`|text|Address of the sign|1577 Bloor Street West
-`sign_name`|text|Name of the sign. May include address + serial number, the ward name for the Mobile WYSP, or school name for Schools WYSP|Dundas St W SB 16101191
-`dir`|text|Direction of the flow of traffic|NB
-
-### Views
-
-`report_dates` contains the list of days, for each sign, where there is more than 10 hours of data between 6:00 and 22:00.
+`locations` contains each ID used for the API, direction of traffic along which the sign is collecting data, and address and name of the sign. Some signs may have information missing from any one of the fields.
 
-|Field name|Data type|Description|Example|
+|Field name|Type|Description|Example|
 |------|------|-------|------|
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
```

Comment:
I remember seeing sign_id as well.  Do you know where and when we use it?
I think api_id is unique to each sign but sign_is is unique to sign and location.
Could you add this to this documentation?

Reply:
It's the same `id` in `wys.locations`. I added a note to the README